src/firefly/html/demo/ffapi-highlevel-test.html

```diff
@@ -121,6 +141,53 @@
             var v2= firefly.makeImageViewer('old2');
             v1.plot(req);
             v2.plotURL('http://web.ipac.caltech.edu/staff/roby/demo/wise-m51-band2.fits');
+
+
+            //-----------------  TABLE DEMO ------------------------//
+            tblReq1 = firefly.util.table.makeIrsaCatalogRequest('wise_allwise_p3as_psd', 'WISE', 'wise_allwise_p3as_psd', null,
+                    {   position: '10.68479;41.26906;EQ_J2000;m31;ned',
+                        SearchMethod: 'Cone',
+                        radius: 300
+                    });
+            tblReq2 = firefly.util.table.makeIrsaCatalogRequest('another wise catalog', 'WISE', 'wise_allwise_p3as_psd', null,
```

Comment:
alt_source will be used if source fail to load.  one use-case is to put a local path to a file in source, and a url to the same file in alt_source.  so, if file is accessible by firefly, it saves the overhead of retrieving the file from a url.